Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -567,6 +567,15 @@ struct DeviceScopes {
     }
 }
 
+// Evaluate the pullback on a one.
+@usableFromInline
+func pullbackOfOneLikeY<T: TensorFlowFloatingPoint, R>(
```

Comment:
`pullbackOfOneLikeY` could use a better name. We can defer this.

Reply:
Acknowledged.